services-manager/src/main/java/uk/gov/gchq/palisade/service/manager/config/TaskConfiguration.java

```diff
@@ -29,15 +29,30 @@
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+/**
+ * The type Task configuration.
+ */

```

Comment:
The JavaDoc comments in this TaskConfiguration class do not explain the methods properly.